### PR TITLE
1.4.5 release notes

### PR DESCRIPTION
**Pull request by howardjohn**
draft until I verify the CNI stuff

### CODE REVIEW

**Comment by ericvn** [comment]
Reminder that when this gets cherry-picked to 1.4 to update the `data/args.yaml`.

File: content/en/news/releases/1.4.x/announcing-1.4.5/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+subtitle: Patch Release
+description: Istio 1.4.5 patch release.
+publishdate: 2020-02-18
+release: 1.4.5
+aliases:
+    - /news/announcing-1.4.5
+---
+
+This release includes bug fixes to improve robustness. This release note describes what’s different between Istio 1.4.3 and Istio 1.4.4.
+
```
Comment (jwendell) [inline content/en/news/releases/1.4.x/announcing-1.4.5/index.md]: 1.4.4 and 1.4.5

File: content/en/news/releases/1.4.x/announcing-1.4.5/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: Announcing Istio 1.4.5
+linktitle: 1.4.5
+subtitle: Patch Release
+description: Istio 1.4.5 patch release.
+publishdate: 2020-02-18
+release: 1.4.5
+aliases:
+    - /news/announcing-1.4.5
```
Comment (ericvn) [inline content/en/news/releases/1.4.x/announcing-1.4.5/index.md]: I'm not sure of the reasons for this alias, and maybe it's something in the doc infrastructure that wants this link to also be active.

Comment (ericvn) [reply content/en/news/releases/1.4.x/announcing-1.4.5/index.md]: Looks like it's a follow on to adding them in all announcements in https://github.com/istio/istio.io/pull/5426.

**Comment by jwendell** [comment]
@howardjohn helm dir doesn't exist anymore :)

**Pull request merged**